ssm-automation/README.md

```diff
@@ -2,7 +2,11 @@
 
 This repository contains example SSM Automation Documents that can be used by SSM [Change Manager](https://docs.aws.amazon.com/systems-manager/latest/userguide/change-manager.html) feature to create PAAs and PAIs using multi-party authorization.
 
-**Note:** This process will require IAM Roles with permissions to access AWS Private CA. Before beginning, please make sure that you have an IAM role with the following trust relationship and permissions:
+## IAM Roles
+This process will require the creation of IAM Roles to approve the Change Manager Templates, Approve the Change Manager Requests, and execute the automation documents. Before beginning, please make sure that you have all of the following IAM roles.
```

Comment:
probably second "Approve" doesn't need to start from a capital letter. Also, maybe we should combine the two into "approve the Change Manager Templates and Requests"?